src/modeler/AutoRest.Swagger/Validation/Core/RulesExtensions.cs

```diff
@@ -64,6 +64,6 @@ public static IEnumerable<Rule> GetValidationRules(this Type type)
         /// <summary>
         /// The collection of rules that apply to all properties regardless of other rules.
         /// </summary>
-        public static IEnumerable<Rule> UniversalRules = new[] { new NoControlCharacters() };
+        public static IEnumerable<Rule> UniversalRules = new List<Rule>();
```

Comment:
The rule does get triggered its just that it never received any faulty strings by the time it was invoked, which makes me believe the AutoRest core engine is rejecting control characters even before the validator is called into action